tools/src/main/python/opengrok_tools/utils/mirror.py

```diff
@@ -361,6 +371,16 @@ def handle_disabled_project(config, project_name, disabled_msg, headers=None,
             run_command(cmd, project_name)
 
 
+def get_mirror_retcode(ignore_errors, value):
+    if ignore_errors:
+        logger = logging.getLogger(__name__)
+        logger.info("error code is {} however '{}' property is true, "
```

Comment:
This message will be displayed even for success. Is that desirable?

Reply:
true, fixed.